tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "inlineSourceMap": true,
     "inlineSources": true,
     "lib": [
-      "es2018"
+      "es2019"
```

Comment:
ah, `Object.fromEntries` is in ES2019 :)

Reply:
yup, some tool in VS Code told me 😁